maropost_api/reports.py

```diff
@@ -0,0 +1,12 @@
+from base import MaropostBase
+
+
+class MaropostReport(MaropostBase):
+
+    def get_bounces(self, **kwargs):
+        response = self.browser.get('/reports/bounces.json', params=kwargs)
+        return self.validator(response).validate()
```

Comment:
I would recommend you add pagination here 
https://github.com/PowerToFly/maropost-api/blob/master/maropost_api/lists.py#L6